docker/roodjongeren_dev/docker-compose.yml

```
@@ -21,8 +21,41 @@ services:
     volumes:
       - ./nginx.conf:/etc/nginx/nginx.conf:ro
     extra_hosts:
-        - "host.docker.internal:host-gateway"
+      - "host.docker.internal:host-gateway"
+  umami_postgres:
+    container_name: umami_postgres
+    image: postgres:14.1
+    environment:
+      - POSTGRES_USER=$UMAMI_DATABASE_USER
+      - POSTGRES_PASSWORD=$UMAMI_DATABASE_PASSWORD
+      - POSTGRES_DB=$UMAMI_DATABASE_USER
+    ports:
+      - '5433:5432'
+    volumes:
+      - umami_volume:/var/lib/postgresql/data
+  umami:
+    container_name: umami
+    build:
+      context: ../umami
+      args:
+        - BASE_PATH=/umami
+        - DATABASE_TYPE=postgresql
+    depends_on:
+      - umami_postgres
+    environment:
+      # Allows us to change the name of the generated JS file
+      # A custom name reduces the chance of ad blockers preventing our analytics from working
+      - TRACKER_SCRIPT_NAME=stasi
+      - COLLECT_API_ENDPOINT=/stasi/knock
+      - DATABASE_URL=postgresql://umami:NC9h7uwqtHGSBr8f@host.docker.internal:5433/umami
+      - BASE_PATH=/umami
+      - PORT=3100
+    ports:
+      - '3100:3100'
```

```suggestion
      - '3100:3100'
     extra_hosts:
      - "host.docker.internal:host-gateway"
```
Needed on Linux systems, similar to above.